### PR TITLE
Remove provider

### PR DESCRIPTION
**Pull request by dutsmiller**
- Remove helm provider (was an antipattern)
- Relocate config template
- Update README with usage example

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -122,30 +113,20 @@ resource "azurerm_key_vault_access_policy" "vault_init" {
 }
 
 module "vault_identity" {
-  source = "/Users/tmiller/gitlab-repos/tfe/terraform-azurerm-kubernetes/identity"
+  source = "/Users/tmiller/gitlab-repos/tfe/terraform-azurerm-kubernetes/aad-pod-identity/identity"
```
Comment (hiscjo01) [inline main.tf:116]: Does this need to change?

File: main.tf [hunk 2]
```diff
@@ -122,30 +113,20 @@ resource "azurerm_key_vault_access_policy" "vault_init" {
...
 }
 
 module "vault_init_identity" {
-  source = "/Users/tmiller/gitlab-repos/tfe/terraform-azurerm-kubernetes/identity"
-
-    kubectl_host                   = var.kubectl_host
-    kubectl_client_certificate     = var.kubectl_client_certificate
-    kubectl_client_key             = var.kubectl_client_key
-    kubectl_cluster_ca_certificate = var.kubectl_cluster_ca_certificate
+  source = "/Users/tmiller/gitlab-repos/tfe/terraform-azurerm-kubernetes/aad-pod-identity/identity"
```
Comment (hiscjo01) [inline main.tf:125]: Should this be ${path.module}

File: main.tf (hunk 1 above)
Comment (dutsmiller) [reply main.tf]: yep, was a remnant from development, thanks

File: main.tf (hunk 2 above)
Comment (dutsmiller) [reply main.tf]: yep, was a remnant from development, thanks

**Pull request merged**